### DISCUSSION TITLE
Moving to Overture Data only

### DISCUSSION THREAD

**Discussion by mtravis** [opengeos/open-buildings#56] [Ideas]
Overture should google buildings soon and as it is now geoparquet by default. I think we should start thinking about how we handle this and possibly look at refactoring the scripts.

**Comment by jbousquin** [comment]
By overture data do you mean moving from s3://us-west-2.opendata.source.coop/cholmes/overture to s3://overturemaps-us-west-2/release? Seems like that would be a good move to shift away from having to maintain the latest data release (https://github.com/OvertureMaps/data). Not sure how #14 , and comment there about making a new repo may or may not relate.
